src/ui/public/styles/base.less

```diff
@@ -577,30 +565,32 @@ fieldset {
 }
 
 .kuiSideBarSection {
-  margin-bottom: 6px;
+  margin-bottom: 10px;
+  background-color: @globalColorWhite;
+  padding: 10px;
+  border-radius: 4px;
+}
+
+.kuiSideBarSection .kuiSideBarSection .kuiSideBarSection{
```

Comment:
Can we either create a class for this and update the markup, or create a brief ticket so we don't forget? I think this one stands out because the solution seems pretty trivial, and the existing selector is potentially pretty confusing for people.